examples/demo/package.yaml

```diff
@@ -63,5 +63,23 @@ package:
           display_name: File
           summary: File to measure
           description: Input file that will be measured
-          type:  file
+          type: file
+          required: true
+    - name: num_chars_api
```

Comment:
I don't think you need a separate function here.  You could just update the existing "num_chars" to take the additional parameter.

Reply:
Bonus points for making the additional parameter do something meaningful as it relates to the file so that it's a slightly more meaningful demo.

Reply:
Agreed. Fixed, although I dropped the duplicate `function` parameter name, since I figured that would be confusing as a demo for a file parameter